### PR TITLE
Fix /queue/all

### PR DESCRIPTION
**Pull request by Skgland**

### CODE REVIEW

File: homu/server.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def result(repo_label, pull):
 
 @get('/queue/<repo_label:path>')
 def queue(repo_label):
-    if repo_label not in g.cfg['repo']:
+    if (repo_label not in g.cfg['repo']) and (repo_label != 'all'):
```
Comment (pietroalbini) [inline homu/server.py:120]: I think the parenthesis could be removed.

**Review by pietroalbini — COMMENTED** [review]
Thanks for sending this PR! Left a small comment.

File: homu/server.py (hunk 1 above)
Comment (Skgland) [reply homu/server.py]: I am not very familiar with the operator precedence in python, but after looking it up I am confident that you are correct and have removed the unnecessary parentheses

**Comment by pietroalbini** [comment]
Thanks!

**Pull request merged**